frontend-cert/html-projects/travel-agency-page/user-stories.md

```diff
@@ -0,0 +1,17 @@
+1. You should have an `h1` element with the text `Discover Italy`.
+
+2. You should have a paragraph below the `h1` introducing the travel opportunities.
+
+3. You should have an `h2` element with the text `Packages`.
+
+4. You should have a `p` element introducing briefly the various packages.
+
+5. You should have an unordered list element with two list items. The two list items should have the text `Group Travels` and `Private Tours`, respectively. The text of each list item should be enclosed by an anchor element.
+
+6. You should have an `h2` element with the text `Top Itineraries`.
+
+7. You should have three `figure` elements, each containing an anchor element and a `figcaption` element. The three anchor elements should have an `img` element as their content.
+
+8. For each `figure` element, use the following image `src` and `figcaption` values respectively: `colosseo.jpg` with `Rome and Center Italy`, `lavaredo.jpg` with `National Parks`, `favignana.jpg` with `South Italy and Islands`.
+
+9. All your anchor elements should have an `href` attribute with the value of `<placeholder>` and a `target` attribute with the value of `_blank`.
```

Comment:
I hope this is clearer. What should we use to replace `<placeholder>`?

Reply:
I think we could use `https://www.freecodecamp.org/learn` for now. I mainly don't like using dead links because it's not how you would do it when actually making a page. An alternative would be to leave it open-ended - "All your anchor elements should have an `href` attribute whose value is a valid URL..."